doc/MONSTERS.md

```diff
@@ -390,37 +392,37 @@ The listed attack types can be as monster special attacks (see "special_attack")
 Makes the monster leap a few tiles. It supports the following additional properties:
 
 ### "max_range"
-(Required) Maximal range to consider for leaping.
+(Required) Maximal range of attack.
 
 ### "min_range"
-TODO: describe this.
+(Required) Minimal range needed for attack.
 
 ### "allow_no_target"
-TODO: describe this.
+Doesn't allow to choose target.
 
 ### "move_cost"
-TODO: describe this.
+Turns needed to complete special attack. 100 move_cost with 100 speed is equal to 1 second/turn.
 
 #### "min_consider_range", "max_consider_range"
-TODO: describe this.
+Minimal range and maximal range to consider for using specific attack.
 
 ## "bite"
-TODO: describe this.
+Makes monster use teeth to bite opponent. Some monsters can give infection by doing so.
 
 ### "damage_max_instance"
-TODO: describe this.
+Max damage it can deal on one bite.
 
 ### "min_mul", "max_mul"
-TODO: describe this.
+How hard is to get free of bite without killing.
 
 ### "move_cost"
-TODO: describe this.
+Turns needed to complete special attack. 100 move_cost with 100 speed is equal to 1 second/turn.
 
 ### "accuracy"
-TODO: describe this.
+How accurate it is.
```

Comment:
What are valid values that can be used for accuracy?

Reply:
Integer. The highest number was 6.
Not many monsters use this one in first place though.